systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1460,6 +1463,69 @@ void checkStorageSizeForVolumes(List<PersistentVolumeClaim> volumes, String[] di
         }
     }
 
+    @Tag(SCALABILITY)
+    @Test
+    void testBigAmountOfTopics() {
+        int numberOfTopics = 300;
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 3, 1).done();
+
+        for (int i = 0; i < numberOfTopics; i++) {
```

Comment:
While it is form of scalability, you are creating the topics in a serial way (create 1 topic -> wait for it to be created -> create next topic). I think it would be good to split this into 2 parts - create the 300 topics first and in second loop check that they were created.

Reply:
Done, but i had to scale down number of topics because verification of one topics now takes 14s, which for 300 topics = 4200s(70m). I scaled to 50.

Reply:
That's because you're calling `describeTopicUsingPodCLI()` for each topic. That method has to spawn a subprocess from the JVM to do a `kubectl exec` into a pod to run `kafka-topics.sh` (i.e. spin up another JVM). It's all hideously inefficient. It would be _much_ better to use Kafka's `AdminClient.describeTopics()` (which you could call _once_ to get the topic info for all the topics you're interested in).